### DISCUSSION TITLE
Running locally, stuck in bootloop unable to authenticate

### DISCUSSION THREAD

**Discussion by simonvroegop** [AntlerVC/firetable#981] [Support / Q&A] [answered]
I'm trying to run Rowy locally. I've followed the Manual Install instructions on [this page](https://docs.rowy.io/setup/install)
I've also followed the instructions on [this page](https://docs.rowy.io/how-to/roles?set-user-roles-tabs=admin-sdk#set-user-roles) to grant myself admin permissions.

Yet whenever I try to boot an instance, I only see an 'Access denied' screen that loads for about half a second before immediately refreshing and loading the same screen.

Does anyone have an idea what I could try to solve this?

https://user-images.githubusercontent.com/28644865/203749205-7717d9b5-98cd-4e47-9b72-ba5220d54caa.mov

**Comment by RajGM** [comment]
Hi @simonvroegop 
Have you added the API keys in the .env file?

**Reply by simonvroegop** [reply]
Yes, they're in there

**Comment by BeeBombshell** [comment, marked as answer]
Hey @simonvroegop, I'm not sure what is causing this error. Could you check for any errors in the console log (using the developer tools `Ctrl` + `Shift` + `i` ) and share a screenshot if possible?
